config/config.go

```diff
@@ -67,11 +67,28 @@ type (
 		MuxTransportName string `yaml:"mux"`
 	}
 
+	DescribeClusterResponseOverrides struct {
```

Comment:
Should we modify failover version increment in ListClusters also? 

When testing shard count changes, [I modified DescribeCluster](https://github.com/temporalio/s2s-proxy/pull/65/files#diff-31dac6f8c5a1a12d19ca5f4ed6a2bdb69d996abaf6958fdcb02dec7957b7d0f5R82-R92) and I found I did not need to change ListClusters. But it might be confusing if DescribeClusters and ListClusters are different, like if we are doing manual debugging.

Reply:
ok, will do in a separate PR.